management/server/file_store.go

```diff
@@ -121,15 +121,25 @@ func restore(file string) (*FileStore, error) {
 			store.PrivateDomain2AccountID[account.Domain] = accountID
 		}
 
-		// if no policies are defined, that means we need to migrate Rules to policies
-		if len(account.Policies) == 0 {
+		// always regenerate policy objects which reflects Rules
```

Comment:
Lets add a Todo here to evaluate policy templating as is the main cause of this regeneration.

Reply:
Fixed